src/client/components/NavBar.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import "../App.css";
+import { Link } from "react-router-dom";
+import logo from "../assets/images/food.png";
+
+function NavBar() {
+  return (
+    <nav>
+      <div>
+        <Link to="/">
+          <img src={logo} style={{ width: "55px", height: "55px" }} />

```

Comment:
with this method, you can reference your local image 
https://stackoverflow.com/questions/39999367/how-do-i-reference-a-local-image-in-react

Reply:
I don't know why but when I put the path directly there the img cannot be shown
However, when I import on the top part and use that is working...so I use that method.

Reply:
you can find the right way down here,  next time try to read the whole documentation carefully
https://github.com/islam-fawzy25/meal-sharing/blob/master/src/client/components/structures/Nav/Nav.js

Reply:
FYI, I also tried other ways to import local images(from your first reference), using 'require' but then I had to change my webpack and it wasn't pleasant to do so I was stuck with using import on the top.